### PR TITLE
#17178 - browser update - sep2018

### PR DESCRIPTION
**Pull request by AnDeVerin**
@podkot @shifteee

### CODE REVIEW

**Review by shifteee — CHANGES_REQUESTED** [review]
You lose some improvements from prev version

File: update.npm.full.js [hunk 1]
```diff
@@ -443,7 +492,6 @@ if (ignorebutton) {
 
 
 op.onshow(op);
-};/*
```
Comment (shifteee) [inline update.npm.full.js:446]: this branch of code should be remover or commented

**Comment by podkot** [comment]
which improvements have been lost? lgtm at the moment

**Comment by AnDeVerin** [comment]
@shifteee 
Commented test call in `update.npm.full.js`

**Review by shifteee — APPROVED** [review]

**Pull request merged**